data/human/intro missions.txt

```diff
@@ -508,3 +508,95 @@ conversation "end of intro missions"
+				goto more
+			
+			label equipment
+			`	Captain Zinaida looks over the <ship> critically. "Well, you need a weapon that does hull damage. A beam laser will get the job done but it will take awhile. The most valuable tool is an asteroid scanner. They're expensive but they connect into your navigation and targeting systems to point you towards minable asteroids. You can also set them to find the most valuable asteroids which increases your profitability per run. After that, a tactical scanner will give you range to target and show how much damage you're doing. An afterburner is good for chasing down fast moving asteroids or getting away from pirates without leaving the system. If you have a fleet, you can target an asteroid and they will all fire on it and then pick up the minerals. You can work much more quickly and harvest more in a run. If you have a carrier, like Bertha there, you can have fighters mine and return the minerals to your hold.`
+			choice
+				`	"I can't fit all that on my ship!"`
+					goto shipmods
+				`	"That all makes sense."`
+					goto more
+			
+			label shipmods
+			`	She gives you a crafty smile before responding. "Not in a factory build, you won't. But you'd be surprised how much space you can free up even in a sparrow if you're willing to flexible about health and safety. Do you really need that much power? If you've got an outfitter on the planet, do you need a hyperdrive, or even a shield regenerator? Maybe a cargo expansion would be more use?" Seeing the look on your face she holds her hands up. "I'm not saying you should, of course, but you have options."`
+				goto more
+
+			label safety
+			`	The captain shrugs and you get the feeling she's disappointed in your question. "Go where pirates aren't welcome," she says. There are some systems in the dirt belt with a strong navy presence and high quality asteroids. Or you could go to the Quarg systems and you won't be bothered. Just know that danger and opportunity go together so you'll often find lucrative mining in uninhabited spaces and around pirate worlds.`
+				goto more
+
+			label pirates
+			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." She shakes her head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo."`
```

Comment:
```suggestion
			`	"You can always bribe them when you get into the system and they'll leave you alone. If the minerals are good enough and you have enough cargo capacity you'll still turn a profit. If you have a fast ship, you can just run away. Most pirates are lazy and will turn back if you head to the edges of the system. Of course, with enough of a fleet you just fight them off." She shakes her head. "Most pirates would be richer and live longer if they just mined instead of extorting bribes and cargo.`
```